torchaudio/csrc/ffmpeg/stream_reader/stream_reader.cpp

```diff
@@ -163,12 +163,42 @@ bool StreamReader::is_buffer_ready() const {
 ////////////////////////////////////////////////////////////////////////////////
 // Configure methods
 ////////////////////////////////////////////////////////////////////////////////
-void StreamReader::seek(double timestamp) {
+void StreamReader::seek(double timestamp, int64_t mode) {
   TORCH_CHECK(timestamp >= 0, "timestamp must be non-negative.");
+  TORCH_CHECK(
+      pFormatContext->nb_streams > 0,
+      "At least one stream must exist in this context");
+
+  AVRational time_base = pFormatContext->streams[0]->time_base;
```

Comment:
What happens if `stream[0]` is non-active (`AVDISCARD_ALL `) stream?

Reply:
I tested it and still returns the correct time base for this stream so the seek will still work.